### PR TITLE
#2 & #7 Pinia, UI and dependency addition

### LINKED ISSUE #2: Show avatars on game page

### LINKED ISSUE #7: Make user info Pinia store fetch properly
Expose the fetch action and call it once the `authUser` login resolves. Within the `HomePage` view

### PR DESCRIPTION
**Pull request by Dawdre**
- Fix Pinia fetching user info properly
- Add avatar to Game View
- Add @vercel/speed-insights package and component

@Ashyy-code this is a bad PR and includes fixes for multiple issues. This is typically bad but 🤷🏻

### CODE REVIEW

File: src/stores/userStore.ts [hunk 1]
```diff
@@ -36,9 +37,10 @@ export const useUserStore = defineStore("user", () => {
     
   }
 
-  if (getUserStorageCredentials.value.ID && getUserStorageCredentials.value.authID) {
-    fetchUser()
-  }
+  // if (getUserStorageCredentials.value.ID && getUserStorageCredentials.value.authID) {
```
Comment (Dawdre) [inline src/stores/userStore.ts:40]: I will delete this but basically you can see this was never true on first load

File: src/views/HomeView.vue [hunk 2]
```diff
@@ -61,6 +61,7 @@ async function submitForm(validationErrors: FormValidationError) {
   } finally {
     if (authenticatedUserResponse.value) {
       userStore.setUserStorageCrendentials(authenticatedUserResponse.value)
+      userStore.fetchUser()
```
Comment (Dawdre) [inline src/views/HomeView.vue:64]: We now call the `fetchUser` Pinia action right after we set local storage and update the store.

**Comment by Ashyy-code** [comment]
Looks good to me :D

**Review by Ashyy-code — APPROVED** [review]
Is this where I approve the review then?

**Pull request merged**